### ISSUE TITLE
Lots and lots of duplication

### ISSUE THREAD

**Issue by cbizon** [ranking-agent/strider#267]
Standup query:
```
query = {
    "message": {
        "query_graph": {
            "nodes": {
                "n0": {
                    "ids": [
                        "NCBIGene:6656"
                    ],
                    "categories": [
                        "biolink:Gene"
                    ]
                },
                "n1": {
		             "categories":[
                    
                         
                         "biolink:BiologicalProcessOrActivity"
                        ]
                },
                "n2": {
                    "ids":["NCBIGene:6657"],
                    "categories":[
                        "biolink:Gene"
                       ]
               }

            },
            "edges": {
                "e0": {
                    "subject": "n0",
                    "object": "n1",
                    "predicates":["biolink:related_to"]

                },
                "e1": {
                    "subject": "n2",
                    "object": "n1",
                    "predicates":["biolink:related_to"]

                }
            }
        }
    }
}
```

2 genes, what processes do they have in common.  We return 1174 results, but only 62 are unique.

Oddly, each result happens either 1, 5, 10, 25, 50, 75, 100, or 150 times.

The most common result (150 copies) is this:
```
{
 "edge_bindings": {
  "e0": [
   {
    "id": "NCBIGene:6656-biolink:participates_in-GO:0006355"
   }
  ],
  "e1": [
   {
    "id": "NCBIGene:6657-biolink:participates_in-GO:0006355"
   }
  ]
 },
 "node_bindings": {
  "n0": [
   {
    "id": "NCBIGene:6656"
   }
  ],
  "n1": [
   {
    "id": "GO:0006355"
   }
  ],
  "n2": [
   {
    "id": "NCBIGene:6657"
   }
  ]
 },
 "score": null
}
```

I suspect KP funkiness but even so, I think we should unique these.  It's possible that there is uniquifiying that could occur in the process, allowing some speedup as well.

**Comment by cbizon** [comment]
The other thing is that this causes trouble for AC, which is trying to count things....

**Comment by cbizon** [comment]
There are also repeated values in D.1.

**Comment by uhbrar** [comment]
This may be fixed via the smarter message merging Alon has been working on implementing. Once that's in, I'll check back to see whether or not that addresses the problem.

**Comment by richakanwar13** [comment]
Can be closed when [#276](https://github.com/ranking-agent/strider/issues/276) is closed.
